src/Microsoft.Windows.CsWin32/templates/PInvokeClassMacros.cs

```diff
@@ -0,0 +1,15 @@
+﻿/// <summary>
+/// This class wrapper is stripped so that individual macros can be requested for generation.
+/// </summary>
+internal class PInvokeClassMacros
+{
+	/// <summary>
```

Comment:
Not sure what the overall policy in this repo, but this file is using tabs, whereas other files appear to be using spaces. Perhaps something is amiss in .editorconfig?

Reply:
Oh wait, I guess there's an .editorconfig file in the templates directory itself that prescribes tabs. Now that I recall, all the code we generate uses tabs, so we want the templates to use tabs as well.

Reply:
That makes sense. I commented because it looked inconsistent :)

Reply:
I'll create a PR to use tabs consistently in the templated files that are using spaces.

Reply:
I pushed as ee1ade7b343f97f210d2fb90262add61a6063a4b

Reply:
🚀